### PR TITLE
#76 set null field cover_image_url if data is null

### LINKED ISSUE #76: [BUGFIX] Cover Image URL invalid
##Issue
When user get list or get detail of phonebook, link url for photo is not get because the link is invalid

## What should the behaviour
User can get valid url for photo in phonebook

## Task
- [x] remove concat url in other package, just concat at endpoint

### PR DESCRIPTION
**Pull request by setiadijoe**
close #76 

- fixing when cover_image_url is null

### CODE REVIEW

File: mocks/testcases/get_detail_phonebook.go [hunk 1]
```diff
@@ -102,7 +102,7 @@ var GetDetailPhonebookData = []GetDetailPhonebook{
 				Village:       location,
 				Latitude:      "-6.231928",
 				Longitude:     "0.988789",
-				CoverImageURL: "http://localhost:9080",
+				CoverImageURL: helper.SetPointerString("http://localhost:9080"),
```
Comment (firmanJS) [inline mocks/testcases/get_detail_phonebook.go:105]: Recomended save to env 'HOST_URL='

**Review by firmanJS — CHANGES_REQUESTED** [review]

File: mocks/testcases/get_detail_phonebook.go (hunk 1 above)
Comment (setiadijoe) [reply mocks/testcases/get_detail_phonebook.go:105]: it is just for testcase, and not as response @firmanJS

Comment (sandisunandar99) [reply mocks/testcases/get_detail_phonebook.go:105]: agree

**Review by sandisunandar99 — APPROVED** [review]

**Review by rindibudiaramdhan — APPROVED** [review]

File: mocks/testcases/get_detail_phonebook.go (hunk 1 above)
Comment (rindibudiaramdhan) [reply mocks/testcases/get_detail_phonebook.go]: why `http://localhost:9080`?
can we change it to `local.test`, if this URL is only for testing and no concern at all.

**Review by firmanJS — APPROVED** [review]

**Pull request merged**